src/Application/Regulation/Command/Period/SavePeriodCommandHandler.php

```diff
@@ -25,7 +25,12 @@ public function __construct(
     public function __invoke(SavePeriodCommand $command): Period
     {
         $command->clean();
-        $command->startDate = $this->dateUtils->mergeDateAndTime($command->startDate, $command->startTime);
+
+        // In the case of a temporary order only
+        if ($command->startTime) {
```

Comment:
Peut-être s'appuyer plutôt sur `isPermanent` ? Et je proposerais de préciser le commentaire, et pourquoi pas le faire en français

```suggestion
        // Les arrêtés permanents n'ont qu'un champ date de début,
        // mais pour les temporaires il faut joindre la date et l'heure
        // dans un même datetime.
        if (!$command->isPermanent) {
```

Reply:
Je suis ok pour le mettre en français, il me semblait que les autres commentaires étaient en anglais c'est pour cette raison que j'ai mis en anglais.

Reply:
Oui, disons qu'il y a un mix grandissant, pour les commentaires un peu techniques / pas évidents à comprendre, perso j'ai tendance à utiliser le français maintenant